triage/utils.py

```diff
@@ -91,6 +93,22 @@ def dt_handler(x):
     ).hexdigest()
 
 
+def retry_if_db_error(exception):
+    return isinstance(exception, sqlalchemy.exc.OperationalError)
+
+
+DEFAULT_RETRY_KWARGS = {
+    'retry_on_exception': retry_if_db_error,
+    'stop_max_delay': 1000,
```

Comment:
I'm sure I can look all this up in the retry module's documentation, but it'd be really awesome to have a few comments here that describe what the end effect of these parameters do.  E.g., what's the unit on `1000`?  Do you have a rough sense of what the sequence of timeouts will look like with the given multiplier?  It needn't be exact.